css-fonts/styles.css

```diff
@@ -0,0 +1,16 @@
+* {
+  font-family: "Poppins", sans-serif;
```

Comment:
good choice, just make sure you include a second font to separate the headings and the text.

Reply:
I used font weight to separate them instead of font family. Do I need another font family?

Reply:
![image](https://user-images.githubusercontent.com/46618757/123888728-f945d080-d908-11eb-9c02-af5a5cfcdd4e.png)